core/app/modules/site-editor/assets/js/module.js

```diff
@@ -161,6 +161,21 @@ export default class SiteEditor {
 					video_url: 'https://www.youtube.com/embed/ACCNp9tBMQg',
 				},
 			},
-			];
+			{
+				type: 'loop',
+				icon: 'eicon-error-404', // this icon needs to be updated
+				title: __( 'Loop', 'elementor' ),
+				urls: {
+					thumbnail: elementorAppConfig.assets_url + '/images/app/site-editor/loop.svg',
+				},
+				tooltip_data: {
+					title: __( 'What is a loop?', 'elementor' ),
+					content: __( 'A Loop Grid is a layout where you can display recurring dynamic content (like listings, portfolios, products, etc.) in a customizable template. How do I create one? 1. You need to start with a master item - this is the template. All of the other instances in the grid will match it. 2. When your master item is ready, connect it to the source of your content. You can do this from the editor panel.', 'elementor' ), // this content needs to be updated
```

Comment:
is it always duplicated?
can we use a source in Core and use it in pro?

Reply:
It is always duplicated. 
In core is a js file and in pro is uses a php file. 

I assume it is best to looking into this for a possible future improvement